### PR TITLE
docs: add Buguide idea and Crime Fiction API project

### PR DESCRIPTION
**Pull request by deesclouds**

### CODE REVIEW

File: data/ideas.json [hunk 1]
```diff
@@ -9,29 +9,29 @@
...
         "level": "intermediate",
-        "tags": "snake,game"
+        "tags": "snake, game"
     },
     {
         "title": "Tetris",
         "description": "Create a Tetris game! Sure, it has been done a million times, but it's really interesting to try to make it!",
         "level": "intermediate",
-        "tags": "tetris,game,classic"
+        "tags": "tetris,game, classic"
```
Comment (ming-tsai) [inline data/ideas.json:18]: Add space before `game`
```suggestion
        "tags": "tetris, game, classic"
```

File: data/projects.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
         "tags": "dungeon, markdown-dungeon, gatsby, reactjs"
     },
     {
-        "name": "Memoy-Card-Game",
-        "display": "Memoy Card Game",
+        "name": "Memory-Card-Game",
```
Comment (ming-tsai) [inline data/projects.json:13]: Rollback this change due to the repository name is `Memoy-Card-Game`
```suggestion
        "name": "Memoy-Card-Game",
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @deesclouds,
I added some suggestions for the pr.

**Review by deesclouds — COMMENTED** [review]
agree with changes for spacing and name issue

**Review by ming-tsai — APPROVED** [review]

File: data/ideas.json [hunk 3]
```diff
@@ -156,21 +156,27 @@
...
         "title": "RGB Slider",
         "description": "A color-generating web app which contains three sliders for Red, Blue and Green colors respectively. On moving the sliders, the background color of the screen will change dynamically according to the RGB values.",
         "level":"basic",
-        "tags": "rgb-slider, slider, color-generato, dynamic-colors"
+        "tags": "rgb-slider, slider, color-generator, dynamic-colors"
+    },
+    {
+        "title": "Buguide",
+        "description": "An insect identifier app",
+        "level": "beginner, intermediate",
```
Comment (ming-tsai) [inline data/ideas.json:179]:
```suggestion
        "level": "intermediate",
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]

File: data/projects.json [hunk 4]
```diff
@@ -248,5 +248,16 @@
...
         "site": "",
         "tags": "score-Keeper, HTML, CSS, JAVASCRIPT"
+    },
+    {
+        "name": "true-crime-api",
+        "display": "Crime Fiction API",
+        "description": "An API that provides additional information based on the obsession with a true crime podcast from the mystery-comedy show ... Only Murders in the Building",
+        "author": "deesclouds",
+        "category": "api, fullstack, web, application, website",
+        "level": "basic, intermediate",
```
[the remark this answers is not in the document]
Comment (ming-tsai) [reply data/projects.json:268]:
```suggestion
        "level": "intermediate",
```

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**